src/Compilers/CSharp/Test/Semantic/Semantics/BindingTests.cs

```diff
@@ -2152,16 +2152,28 @@ public void GenericAttributeClassWithMultipleParts()
             var source =
 @"class C<T> { }
 class C<T> : System.Attribute { }";
-            CreateCompilation(source).VerifyDiagnostics(
+            CreateCompilation(source, parseOptions: TestOptions.Regular9).VerifyDiagnostics(
                 // (2,7): error CS0101: The namespace '<global namespace>' already contains a definition for 'C'
                 // class C<T> : System.Attribute { }
-                Diagnostic(ErrorCode.ERR_DuplicateNameInNS, "C").WithArguments("C", "<global namespace>"),
-                // (2,14): error CS0698: A generic type cannot derive from 'System.Attribute' because it is an attribute class
+                Diagnostic(ErrorCode.ERR_DuplicateNameInNS, "C").WithArguments("C", "<global namespace>").WithLocation(2, 7),
+                // (2,14): error CS8773: Feature 'generic attributes' is not available in C# 9.0. Please use language version 10.0 or greater.
                 // class C<T> : System.Attribute { }
-                Diagnostic(ErrorCode.ERR_GenericDerivingFromAttribute, "System.Attribute").WithArguments("System.Attribute")
+                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion9, "System.Attribute").WithArguments("generic attributes", "10.0").WithLocation(2, 14)
                 );
         }
 
+        [Fact]
+        public void GenericAttributeClassWithMultiplePartsWithGenericAttributeFeature()
```

Comment:
>GenericAttributeClassWithMultiplePartsWithGenericAttributeFeature [](http://example.com/codeflow?start=20&length=65)

Consider merging with the previous test.

Reply:
Will do.